static/js/timer.js

```diff
@@ -1,6 +1,24 @@
 var timer = document.getElementById('timer');
+var totaltime = document.getElementById('totaltime');
 var title = document.getElementsByTagName("title")[0];
 
+var timerVar = setInterval(countTimer, 1000);
+var totalSeconds = 0;
+
+function countTimer() {
+  if(isPause){
+    return;
+  }
+  if(totaltimectr){
+    return;
+  }
+  ++totalSeconds;
+  var hour = Math.floor(totalSeconds / 3600);
+  var minute = Math.floor((totalSeconds - hour*3600) / 60);
+  var second = totalSeconds - (hour*3600 + minute*60);
+  document.getElementById("totaltime").innerHTML = ('0' + hour).substr(-2) + ":" + ('0' + minute).substr(-2) + ":" + ('0' + second).substr(-2);
```

Comment:
1. Should reuse variable `totalTime` above 
2. Please incorporate `moment.js` or something similar to make time calculation a breeze and less error prone.